business/MetadataDatabase/Convertor/UserRegisterConvertor.cs

```diff
@@ -0,0 +1,35 @@
+﻿using MetadataDatabase.Data;
+using MetadataDatabase.Models;
+using MongoDB.Bson;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MetadataDatabase.Convertor
+{
+    public static class UserRegisterConvertor
+    {
+        public static User ToModel(this UserRegisterDto userRegisterDto)
+        {
+            return new User
+            {
+                Id = userRegisterDto.Id.ToObjectId(),
+                lastname = userRegisterDto.lastname,
+                firstname = userRegisterDto.firstname,
+                login = userRegisterDto.login,
+                role = userRegisterDto.role,
+                password = userRegisterDto.password,
+            };
+        }
+        public static UserDto ToDto(this UserRegisterDto user)
```

Comment:
The ToDto should take a User not a UserDto

Reply:
mumh I rename ToDto to ToUserDto because I need to transform a UserRegisterDto to UserDto for update